ConwaysGameOfLife/GameController.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+
+namespace ConwaysGameOfLife
+{
+    public class GameController
+    {
+        // responsible for implementing game
+        // orchestrator type class - brings all together
+
+        /*
+            welcome message
+            user asked for universe dimensions
+            dimensions parsed
+            universe initialized
+            initial universe printed to console
+            user asked for location of alive cells
+            input parsed/validated
+            universe set up with live cells
+            universe printed
+            game plays
+            rules checked
+            new universe regenerated
+            ends either by user typing q or all dead cells
+
+            input
+            output
+            universe
+            rules
+        */
+        private Universe _universe;
+
+        private IOutput _output;
+
+        private IInput _input;
+
+        public GameController(Universe universe, IOutput output, IInput input)

```

Comment:
Are you sure you want a `Universe` being passed into the `GameController`? Would the `GameController` not make the very first `Universe` from the `IInput`?

A nitpick, but input/output generally is shortened to I/O - could consider having `input` as the first parameter and `output` as the second.
```suggestion
        public GameController(IInput input, IOutput output)
```